kubert/src/server.rs

```diff
@@ -320,10 +322,10 @@ impl TlsKeyPath {
     async fn load_private_key(&self) -> std::io::Result<rustls::PrivateKey> {
         // Open keyfile.
         let pem = tokio::fs::read(&self.0).await?;
-        let mut reader = std::io::BufReader::new(pem.as_slice());
 
         // Load and return a single private key.
-        let keys = rustls_pemfile::rsa_private_keys(&mut reader)?;
+        let keys = rustls_pemfile::pkcs8_private_keys(&mut BufReader::new(pem.as_slice()))
+            .or_else(|_| rustls_pemfile::rsa_private_keys(&mut BufReader::new(pem.as_slice())))?;
```

Comment:
do we want to maybe log something about which format the keys were parsed in?

Reply:
I could go either way. for now I'll trust that whatever rustls logs is good enough